lib/Spreadsheet/XLSX.rakumod

```diff
@@ -54,10 +62,11 @@ class Spreadsheet::XLSX does Spreadsheet::XLSX::Root {
             }
 
             # Locate the root relationships file, and using it, the workbook root.
-            with self.find-relationships('') -> Spreadsheet::XLSX::Relationships $top-rel {
-                with $top-rel
+            with $!root-relationships = self.find-relationships('') {
+                with $!root-relationships
```

Comment:
Why the nested `with`? Given the assignment on the previous line, it's going to be the same outcome as the outer `with`?

Reply:
Perhaps an overlooked leftover. Will look into it later today as well as into fixing the conflict.

Reply:
Ah, no, there is no problem here. The nested `with` is for a chain of method calls `$!root-relationships.find-by-type(...).first`. `find-by-type` argument is a long schema URI, so the chain is split into multiple lines for readability.